portfolio/src/main/webapp/src/components/theme-switcher.ts

```diff
@@ -0,0 +1,39 @@
+import * as Surplus from 'surplus';
```

Comment:
Clean up stale surplus imports.

Reply:
Fixed in future PR.

Reply:
I'd prefer you not add something just to remove it in a future PR. Please remove it here. Same with the set function mentioned in another comment. In general, it's fine to punt a refactor or feature request to a future PR, but cleanups like this should be addressed sooner.

Reply:
Okay